src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -384,7 +384,7 @@ public static ListBoxModel listCheckoutCredentials(@CheckForNull Item context, S
             GitHubConnection.connect(
                 connectionId, gb.build(), cache, credentials instanceof GitHubAppCredentials);
       }
-
+      
```

Comment:
```suggestion
```

Reply:
Heh, that's what I get for not running formatting check before committing. 😄